specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachineScaleSet.json

```diff
@@ -3925,6 +3931,18 @@
             "lun"
           ],
           "description": "Specifies the parameters that are used to add data disks to the virtual machines in the scale set. <br><br> For more information about disks, see [About disks and VHDs for Azure virtual machines](https://docs.microsoft.com/azure/virtual-machines/managed-disks-overview)."
+        },
+        "diskControllerType": {
+          "type": "string",
+          "description": "Specifies the disk controller type configured for the virtual machines in the scale set. This property is only supported for virtual machines whose operating system disk and and VM size supports hyperVGeneration: V2. For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes. <br> NOTE: You need to deallocate the virtual machines in the scale set before updating its disk controller type based on the upgrade mode configured for the scale set.<br><br> Minimum api-version: 2020-08-01",
+          "enum": [
+            "SCSI",
+            "NvMe"
+          ],
+          "x-ms-enum": {
```

Comment:
that is correct.  If this is just a virtualMachine property, then we can keep this here and remove the one in ComputeRPCommon.json.  If this will be also used in VMSS at some point or other resources in compute, then let's leave it in ComputeRPCommon.json and refer to that definition here @kimberlyyipc  @raktdas

Reply:
Got it..will address